### PR TITLE
DOC: Improvements on logging

### PR DESCRIPTION
**Pull request by willianpaixao**
Is this feature desired, fellow maintainers?

When cleaning old bundles, I had no idea which ones were being deleted and whether it was successful or not.
A few logging lines would be appreciated.

### CODE REVIEW

**Comment by freddiev4** [comment]
Hey @willianpaixao thanks for the PR!

Could you just post an an example output for when you clean the bundles up?

**Comment by willianpaixao** [comment]
Here you go:

``` bash
$ zipline bundles
csvdir <no ingestions>
quandl 2019-11-10 00:45:58.577633
quandl 2019-11-10 00:34:29.738155
quandl 2019-11-10 00:26:38.237694

$ zipline clean --keep-last 1
[2019-11-10 00:53:44.868754] INFO: zipline.data.bundles.core: Cleaning 2019-11-10T00;26;38.237694.
[2019-11-10 00:53:44.909732] INFO: zipline.data.bundles.core: Cleaning 2019-11-10T00;34;29.738155.
```

I've just rebased my branch to upstream/master to check if there won't be any problems merging an six months old branch. All tests passed.

I'm also removing the WIP tag since I don't see any more logs to be added in this specific command (`zipline clean`). If you find another place to add more log messages, I would be happy to implement.

**Comment by willianpaixao** [comment]
Hey @freddiev4 @richafrank can we get this PR merged please?

File: zipline/data/bundles/core.py [hunk 1]
```diff
@@ -432,6 +435,7 @@ def ingest(name,
                     raise ValueError('Need to ingest a bundle that creates '
                                      'writers in order to downgrade the assets'
                                      ' db.')
+            log.info("Ingesting %s." % name)
```
Comment (ssanderson) [inline zipline/data/bundles/core.py]:
```suggestion
            log.info("Ingesting {}.", name)
```

File: zipline/data/bundles/core.py [hunk 2]
```diff
@@ -605,6 +609,7 @@ def should_clean(name):
         cleaned = set()
         for run in all_runs:
             if should_clean(run):
+                log.info("Cleaning %s." % run)
```
Comment (ssanderson) [inline zipline/data/bundles/core.py]:
```suggestion
                log.info("Cleaning {}.", run)
```

**Comment by willianpaixao** [comment]
Suggestions applied, @ssanderson.

**Comment by ssanderson** [comment]
thanks @willianpaixao

**Pull request merged**